### DISCUSSION TITLE
Can't get most basic multi env to work.

### DISCUSSION THREAD

**Discussion by soemarko** [goatandsheep/react-native-dotenv#385] [Q&A] [answered]
I just need the most basic setup, but can't seem to get it up and running. My setup:

- I have 2 env files: `.env` and `.env.production`
- babel.config.js: `plugins: [ ["module:react-native-dotenv"] ]`
- the code: `import {API_URL} from '@env'; console.log(API_URL);`

Regardless of how I run the project, it only spits out the .env values. What I've tried:

- `APP_ENV=production npx react-native run-ios`
- `NODE_ENV=production npx react-native run-ios`
- `APP_ENV=production npx react-native start` then run the ios from separate terminal window
- `NODE_ENV=production npx react-native start` same as above.

Did I miss anything obvious?

**Comment by soemarko** [comment, marked as answer]
Figured it out. Needed to run start --resetCache in between (and sometimes, needed to re-start a couple more times for the changes to pickup).

A bit annoying since run-ios/android doesn't pass through --resetCache, this doesn't work with my flow. And at this point `APP_ENV=production yarn start --resetCache` might as well becomes `cp .env.production .env && yarn start --resetCache`.

A simple config.js file may fit me better at this point. Maybe i'll revisit this plugin in the future. Thanks!

**Reply by goatandsheep** [reply]
for now! this is also a sign that the `addExternalDependencies` feature is not working for you. I really hope that once we get #377 straightened out that you won't need to clear the cache by hand
